Wikipedia/Code/WMFContinueReadingContentSource.m

```diff
@@ -34,7 +34,7 @@ - (void)stopUpdating {
 }
 
 - (void)loadNewContentInManagedObjectContext:(NSManagedObjectContext *)moc force:(BOOL)force completion:(nullable dispatch_block_t)completion {
-    NSURL *lastRead = [moc wmf_openArticleURL] ?: [self.userDataStore.historyList mostRecentEntryInManagedObjectContext:moc].URL;
+    NSURL *lastRead = [self.userDataStore.historyList mostRecentEntryInManagedObjectContext:moc].URL;
```

Comment:
should we get rid of `wmf_openArticleURL` completely and use `mostRecentEntryInManagedObjectContext` in other places? 🤔 https://github.com/wikimedia/wikipedia-ios/blob/27cd80b3aacdc36cbc3d541006337f318a4185d1/Wikipedia/Code/WMFAppViewController.m#L1505

Reply:
I think there's still a useful differentiation between `wmf_openArticleURL` and `mostRecentEntryInManagedObjectContext` - the former is an article that's open and being actively read, the latter is just the most recently read article, regardless of whether or not it's open. Either way, I think I should revert the removal of `wmf_openArticleURL` here and we can discuss that separately